Classes/View/Form.php

```diff
@@ -439,39 +439,39 @@ protected function fillDefaultMarkers()
         }
         $markers['###HIDDEN_FIELDS###'] = '
 			<input type="hidden" name="id" value="' . $GLOBALS['TSFE']->id . '" />
-			<input type="hidden" name="' . $name . '" value="1" />
+			<input type="hidden" name="' . htmlspecialchars($name) . '" value="1" />
```

Comment:
Why hsc this? FormValuesPrefix is a formhandler setting and isn't influenced by post/get vars?